content/en/docs/zero-code/python/troubleshooting.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Troubleshooting Python automatic instrumentation issues
+linkTitle: Troubleshooting
+weight: 40
+aliases: [/docs/languages/python/automatic/troubleshooting]
```

Comment:
Aliases are (mainly) used for backwards support of old links. This is a new page, so no need for an alias:

```suggestion
```

Reply:
Ah thanks! Committed in [2c21d0e](https://github.com/open-telemetry/opentelemetry.io/pull/7036/commits/2c21d0ea9f36d7be0ea5c8e2fc582fe024887d84)